proposals/0069_prometheus-exporters-otel-embedding.md

```diff
@@ -0,0 +1,286 @@
+## Embeddable Prometheus Exporters in OpenTelemetry Collectors
+
+* **Owners:**
+  * [@ArthurSens](https://github.com/ArthurSens)
+
+* **Implementation Status:** Not implemented
+
+* **Related Issues and PRs:**
+  * https://github.com/prometheus/exporter-toolkit/pull/357
+  * https://github.com/prometheus/node_exporter/pull/3459
+  * https://github.com/ArthurSens/prometheus-opentelemetry-collector (proof of concept)
+
+* **Other docs or links:**
+
+> TL;DR: This proposal introduces a mechanism to embed Prometheus exporters as native OpenTelemetry Collector receivers, reducing duplication of effort between the two ecosystems and enabling the "single binary" promise for telemetry collection without forcing reimplementation of hundreds of existing Prometheus exporters.
+
+## Why
+
+The OpenTelemetry Collector ecosystem faces a significant challenge: many components in collector-contrib are "drop-in" replacements for existing Prometheus exporters but often become unmaintained before reaching stability. This duplication of effort occurs because the promise of "one binary to collect all telemetry" is valuable to users, leading to reimplementation of functionality that already exists in mature Prometheus exporters.
```

Comment:
> "one binary to collect all telemetry" is valuable to users

If you ask me, it's a pretty bad idea for the users, or in other words: it has a net-negative value to users.

But obviously, this discussion is out of scope for this design doc. I propose to not value the approaches here. Just state that this approach is the one taken in the OTel collector ecosystem, without valuing it either way.